oneview_redfish_toolkit/authentication.py

```diff
@@ -103,6 +102,15 @@ def get_multiple_oneview_token():
 
 def create_credentials(username, password):
     credentials = dict()
+
+    try:
+        login_domain, username = username.split("\\")
+    except (AttributeError, ValueError):
```

Comment:
I would change the message to something like  "AuthLoginDomain not specified on session creation", otherwise it seems an error.